docs/releases/5.1.txt

```diff
@@ -289,7 +289,8 @@ Requests and Responses
 Security
 ~~~~~~~~
 
-* ...
+* :class:``~django.contrib.auth.hashers.ScryptPasswordHasher`` now has an increased 
+  default value of 5 for the ``parallelism`` parameter to meet OWASP recommendations. 
```

Comment:
```
* In order to follow OWASP recommendations, the default ``parallelism`` of the
  :class:``~django.contrib.auth.hashers.ScryptPasswordHasher`` is increased to 5.
```
Please move this to the `django.contrib.auth` section.

Reply:
I appreciate your guidance, and I've now moved to that specific part.